sbws/lib/v3bwfile.py

```diff
@@ -432,6 +435,42 @@ def add_stats(self, **kwargs):
         [setattr(self, k, str(v)) for k, v in kwargs.items()
          if k in STATS_KEYVALUES]
 
+    def add_relays_excluded_counters(self, exclusion_dict):
```

Comment:
Hm, who is calling this func?

Reply:
hmm, i think this might be coming from other ticket/branch and added here by mistake, will check

Reply:
yes, it was added in https://github.com/torproject/sbws/pull/348/commits/5fd38798a8070818d0b521b56b5ca50943af4ca7#diff-0c88fa36ad3aabea4aa3c57e63f54db8R441
(i should work on less tickets at a time).